crengine/src/lvtinydom.cpp

```diff
@@ -9056,8 +9056,14 @@ ldomXPointer ldomDocument::createXPointer( lvPoint pt, int direction, bool stric
                 }
 
                 lUInt32 hints = WORD_FLAGS_TO_FNT_FLAGS(word->flags);
-                font->measureText( str.c_str()+word->t.start, word->t.len, width, flg, word->width+50, '?',
-                            src->lang_cfg, src->letter_spacing + word->added_letter_spacing, false, hints);
+                if (str.empty() && word->t.len > 0) {
+                    // Don't know that the fuck up, but it happens
+                    for (int i = 0; i < word->t.len; i++)
+                        width[i] = 0;
```

Comment:
Strange indeed.
Can you reproduce this? Specific book? Specific action that uses createXPointer() ?

Reply:
> Can you reproduce this? Specific book?

Yes, of cource, book: http://fred-wang.github.io/mathml.css/

I don't know how to reproduce this in KOReader, but in the desktop version of CoolReader, just move the mouse over the text and the program will crash sooner or later.
Just see backtrace in commit comment.

Reply:
Well, can't reproduce. Anyway, it feels that if this happens, the bug might be elsewhere, and it feels awkward handling that here.
If you have the time, could you try to printf the xpointer.toStringV1() when this happens, and try to find out what bit of HTML/MathML is involved in this document?

Just curious: why is there such a CR3View::mouseMoveEvent() handler ? Does it really get the xpointer at each mouse move ?! Feels it might feel really slow.

Reply:
> Just curious: why is there such a CR3View::mouseMoveEvent() handler ? Does it really get the xpointer at each mouse move ?! Feels it might feel really slow.

Sorry, I have zero interes to dig this depper.